sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/ExtractPrebuiltReceiptAsync.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.ExtractedReceipt;
+import com.azure.ai.formrecognizer.models.OperationResult;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.IterableStream;
+import com.azure.core.util.polling.PollerFlux;
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+
+/**
+ * Sample for extracting receipt information using file source URL.
+ */
+public class ExtractPrebuiltReceiptAsync {
+    public static void main(final String[] args) {
+        // Instantiate a client that will be used to call the service.
+        final FormRecognizerAsyncClient client = new FormRecognizerClientBuilder()
+            .apiKey(new AzureKeyCredential(""))
+            .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+            .buildAsyncClient();
+
+        String receiptUrl = "https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/media/contoso-allinone.jpg";
+        PollerFlux<OperationResult, IterableStream<ExtractedReceipt>> analyzeReceiptPoller =
+            client.beginExtractReceipt(receiptUrl, true, Duration.ofSeconds(1));
+
+        IterableStream<ExtractedReceipt> receiptPageResults = analyzeReceiptPoller
+            .last()
+            .flatMap(trainingOperationResponse -> {
+                try {
+                    Thread.sleep(20000);
```

Comment:
Why do we sleep here for 20 seconds? The captured `trainingOperationResponse` won't change during the sleep period.